pkg/operator/operators/clustermanager/storageversionmigration/precheck.go

```diff
@@ -0,0 +1,125 @@
+package storageversionmigration
+
+import (
+	"context"
+	"strings"
+
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog/v2"
+	migrationv1alpha1 "sigs.k8s.io/kube-storage-version-migrator/pkg/apis/migration/v1alpha1"
+	migrationv1alpha1client "sigs.k8s.io/kube-storage-version-migrator/pkg/clients/clientset/typed/migration/v1alpha1"
+)
+
+var checkList []string = []string{
+	"managedclustersets.cluster.open-cluster-management.io",
+	"managedclustersets.v1beta1.cluster.open-cluster-management.io",
+	"managedclustersetbindings.cluster.open-cluster-management.io",
+	"managedclustersetbindings.v1beta1.cluster.open-cluster-management.io",
+}
```

Comment:
Only the following two migration CRs should be checked:
- "managedclustersets.v1beta1.cluster.open-cluster-management.io"
- "managedclustersetbindings.v1beta1.cluster.open-cluster-management.io"

Reply:
Removed the other two migrations.